OpenRA.Game/Map/Map.cs

```diff
@@ -450,7 +450,10 @@ void PostInit()
 			}
 
 			Sequences = new SequenceSet(this, modData, Tileset, SequenceDefinitions);
-			Translation = new Translation(Game.Settings.Player.Language, Translations, this);
+			Translation = new Translation(Game.Settings.Player.Language,
```

Comment:
We shouldn't bother creating a new `Translation` object if `TranslationDefinitions == null` - this ends up being redudant against the `TranslationProvider`. Just assign it null.

Reply:
Changed it anyway